zulip/zulip/__init__.py

```diff
@@ -1554,6 +1554,53 @@ def set_typing_status(self, request: Dict[str, Any]) -> Dict[str, Any]:
             request=request
         )
 
+    def move_topic(
+        self,
+        stream: str,
+        new_stream: str,
+        topic: str,
+        new_topic: Optional[str] = None
+    ) -> Dict[str, Any]:
+        """Move a topic from ``stream`` to ``new_stream``
+
+        The topic will be renamed if ``new_topic`` is provided
+
+        Example usage::
+
+            >>> client.move_topic('stream_a', 'stream_b', 'my_topic')
+            {'result': 'success', 'msg': ''}
+        """
+        # ask the server for the latest message ID in the topic.
+        request = {
+            'anchor': 'newest',
+            'narrow': [{'operator': 'stream', 'operand': stream},
+                       {'operator': 'topic', 'operand': topic}],
+            'num_before': 1,
+            'num_after': 0,
+        }
+        reply = self.get_messages(request)
```

Comment:
Let's call this `result`; we use that local variable name for API results in the codebase.

Also, can you add a few blank lines in this function to match the style of the codebase?